frontend-react/src/pages/admin/value-set-editor/ValueSetsIndex.tsx

```diff
@@ -31,41 +26,36 @@ const Legend = ({ items }: { items: LegendItem[] }) => {
         </section>
     );
 };
-const sampleValueSetColumnConfig: ColumnConfig[] = [
+const valueSetColumnConfig: ColumnConfig[] = [
     {
-        dataAttr: "value",
-        columnHeader: "Value",
+        dataAttr: "name",
+        columnHeader: "Valueset Name",
     },
     {
-        dataAttr: "header",
-        columnHeader: "Column header",
+        dataAttr: "system",
+        columnHeader: "System",
     },
     {
-        dataAttr: "type",
-        columnHeader: "Value type",
-        editable: true,
+        dataAttr: "createdBy",
+        columnHeader: "Created By",
+    },
+    {
+        dataAttr: "createdAt",
+        columnHeader: "Created At",
     },
 ];
-/* END OF FAUX DATA AND STUFF TO BE REMOVED WHEN IMPLEMENTING THE API */
+const useValueSetsTable = generateUseLookupTable<ValueSet>(
+    LookupTables.VALUE_SET
+);
 
 const ValueSetsTable = () => {
-    /* This would be replaced by our API response as reactive state (useResource) */
-    const [sampleValueSetArray, setSampleValueSetArray] = useState<ValueSet[]>([
-        {
-            value: "Patient ID",
-            header: "patient_id",
-            type: "Unique identifier",
-        },
-        {
-            value: "Patient last name",
-            header: "patient_last_name",
-            type: "Text",
-        },
-    ]);
-    /* We'd pass our config and our API response in this */
+    const valueSetArray = useValueSetsTable();
+
+    const [, setValueSet] = useState<ValueSet[]>([]);
```

Comment:
Is this an artifact of the linter? Can't imagine we'd have a use case for setting a state variable without ever using it 😆

Reply:
@kevinhaube the linter guided us there, but the real reason is that it is not actually being used at the moment (no updates on this page for MVP). I left that there because the Details page will need this to clone from